descartes_light/core/include/descartes_light/solvers/ladder_graph/ladder_graph.h

```diff
@@ -190,11 +253,16 @@ class LadderGraph
         Node<FloatType> node2 = rung2.nodes.front();
         typename State<FloatType>::ConstPtr state2 = node2.sample.state;
         out << "Rung # " << failed_id << "\n";
+        out << "\tNode 1\t|\tNode 2\t|\tDiff (Node2 - Node1)\n";
```

Comment:
I would also move this functions implementation to `ladder_graph.hpp`

Reply:
I'm having difficulties doing that. Since the `<<` operator is defined as a `friend` object it leads to weird things, and I can't figure out how to accomplish moving it

Reply:
Okay, lets leave this here for now.